rpms/pulp-scl/pulp-scl.spec

```diff
@@ -0,0 +1,153 @@
+# We specificall separate the 'pulp' and '-scl' parts from each other.
+# since spec file macros don't allow '-' and the auto generated ones
+# are based on %scl_name
+%global scl_name_base pulp
+%global scl_name_postfix -scl
+
+%global scl_vendor pulp
+%global _scl_prefix /opt/%{scl_vendor}
+
+# scl specific macros
+%global scl %{scl_name_base}%{scl_name_postfix}
+%scl_package %scl
+
+# Python scl macros
+%{!?scl_python:%global scl_python rh-python35}
+%{!?scl_prefix_python:%global scl_prefix_python %{scl_python}-}
+
+# Only for this build, you need to override default __os_install_post,
+# because the default one would find /opt/.../lib/python2.7/ and try
+# to bytecompile with the system /usr/bin/python2.7
+%global __os_install_post %{%{scl_no_vendor}_os_install_post}
+# Similarly, override __python_requires for automatic dependency generator
+%global __python_requires %{%{scl_no_vendor}_python_requires}
+
+# Generate our sitelib variable from %scl_python's sitelib variable.  
+# Secifically, we replace /rh/%{scl_python} with /pulp/pulp-scl/
+%global pulp_sitelib %(echo %{python35python_sitelib} | sed 's|/rh/%{scl_python}|/%{scl_vendor}/%{scl}|')
+
+%global debug_package %{nil}
+%global nfsmountable 1
+
+Name:		%scl_name
+Version:	0.1
+Release:	12%{?dist}
```

Comment:
For package builds, it is not easy to 'replace' the same version with a freshly built package.  RPM spec files provide various ways to specify this package is an update.

The easiest way (and the way that is used with modifications to the .spec file), is to bump the 'Release' field, which basically states it's the same upstream package, but we are releasing a new 'Release' of the downstream package.

In this specific example, this package has been built successfully in copr 12 times.  After each successful build, I have to update the release number to super-cede the previous one.  I'm keeping the Version field at a 0.x until such time that we are satisfied that the pulp-scl itself finally hits all the requirements.  When we release 'Version' 1.0, 'Release' will drop back down to 1